akka-docs/rst/java/http/client-side/websocket-support.rst

```diff
@@ -83,4 +87,31 @@ underlying TCP interface. The same scenarios as described for regular HTTP reque
 
 The returned layer forms a ``BidiFlow<Message, SslTlsOutbound, SslTlsInbound, Message, CompletionStage<WebSocketUpgradeResponse>>``.
 
+.. _half-closed-client-websockets-java:
+
+
+Half-Closed WebSockets
+----------------------
+The WebSocket API does not support half-closed connections, which means that if the either stream completes, the
```

Comment:
I think the problem is not on protocol level but with our API. (At least that is how I understood Johannes in the ticket.)